### ISSUE TITLE
I can't upload files more than 10M

### ISSUE THREAD

**Issue by Eronana** [tenox7/wfm#53]
I just found WFM, it is simple, nice and great, but I can't upload files more than 10M.
It seems 10M hardcoded here, it's better to specify it with a flag.
https://github.com/tenox7/wfm/blob/0b180b15bca93dcb47ec1eafb1df6813b27b5f3f/handlers.go#L27

**Comment by tenox7** [comment]
ah good idea, I can add a flag or you can send me a PR if you want

**Comment by Eronana** [comment]
Another idea is to get rid of forms, use a separate endpoint for uploads.
The frontend just posts the raw file data, and the server writes request body to a file directly, no content needs to be parsed, no buffer needed.

**Comment by tenox7** [comment]
sounds very interesting, do you have an example?

**Comment by tenox7** [comment]
added a flag `-form_maxmem=xxx`

**Comment by tenox7** [comment]
did a little bit of research, what you're asking with the separate handler is possible with some caveats, I don't think that you can get rid of forms as such, but it would probably be possible to use streaming processing of mime/multipart

**Comment by Eronana** [comment]
Sorry for the late reply, I tried to implement it today, I implemented the back-end part, and then when I tried to write the front-end, I realized that the project doesn't use JavaScript, which means I can't use xhr or fetch to POST file data.
And the form doesn't support sending just the raw file data directly, which means my idea doesn't work.

All in all, `form_maxmem` is the perfect solution for now!
